components/raftstore/src/store/fsm/peer.rs

```diff
@@ -4778,9 +4785,9 @@ where
 
     fn on_entry_cache_evict_tick(&mut self) {
         fail_point!("on_entry_cache_evict_tick", |_| {});
-        if needs_evict_entry_cache(self.ctx.cfg.evict_cache_on_memory_ratio) {
-            self.fsm.peer.mut_store().evict_cache(true);
-        }
+        // call evict directly here, no need to recheck the memory usage condition.
```

Comment:
why

Reply:
Because we only register the reclaim tick when memory is insufficient.

Reply:
@tier-cap I don't know the details, just a question: 

Is it possible that between registration and response, other parts of memory have been reclaimed, at this point there has been enough memory to not need evict?

Reply:
right, we only evict at most applied entries, not all.